### ISSUE TITLE
Utilize a contextual menu rather than a custom menu.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [dfaust/plasma-applet-popup-launcher#13]
"http://user-images.githubusercontent.com/42837531/174346714-973c7e30-cf88-4b78-ab5a-a835900b7498.png" depicts what it should be, to ease maintenance and adhere to the customization of the host operating-system. This improves accessibility and consistency.

Please know that this obviously does not prevent presentation of iconography. I have merely disabled it for myself.
